src/lib/cache/gdfs_cache.hpp

```diff
@@ -28,6 +28,36 @@ class GDFSCache : public AbstractCacheImpl<Key, Value> {
   };
 
   using Handle = typename boost::heap::fibonacci_heap<GDFSCacheEntry>::handle_type;
+  using CacheMap = typename std::unordered_map<Key, Handle>;
+
+  using typename AbstractCacheImpl<Key, Value>::KeyValuePair;
```

Comment:
Didn't know you could do that

Reply:
Just a side note or do you think that's something I should better change?

Reply:
just a note